pkgs/top-level/all-packages.nix

```diff
@@ -577,6 +577,8 @@ with pkgs;
 
   avfs = callPackage ../tools/filesystems/avfs { };
 
+  avldrums.lv2 = callPackage ../applications/audio/avldrums.lv2 { };
```

Comment:
you cannot have a dot in an attribute name because nix will interpret that as the attibute `lv2` of the set `avldrums`, which doesn't make sense (that's why ofborg says zero builds). Please rename consistently to `avldrums-lv2` (or another acceptable name).